README.rst

```diff
@@ -20,11 +20,13 @@
 scrapy-fake-useragent
 =====================
 
-Random User-Agent middleware based on
-`fake-useragent <https://pypi.python.org/pypi/fake-useragent>`__. It
-picks up ``User-Agent`` strings based on `usage
-statistics <http://www.w3schools.com/browsers/browsers_stats.asp>`__
-from a `real world database <http://useragentstring.com/>`__.
+Random User-Agent middleware for Scrapy scraping framework based on
+`fake-useragent <https://pypi.python.org/pypi/fake-useragent>`__, which picks up ``User-Agent`` strings 
+based on `usage statistics <http://www.w3schools.com/browsers/browsers_stats.asp>`__
+from a `real world database <http://useragentstring.com/>`__ and 
```

Comment:
I'd keep the phrase that we get real-world user agent strings but also say that fake generated user agent could be configured as a backup option.